### ISSUE TITLE
(Item Request) Suspended Cables

### ISSUE THREAD

**Issue by pooing1234** [Electrical-Age/ElectricalAge#730]
Something like this http://imgur.com/a/3BSJ9
To easily enable connections between 2 buildings.

Edit: this could also allow for custom power poles.

**Comment by OmegaHaxors** [comment]
That looks pretty good for a paint job. Maybe this could be a possible implementation of the power plugs?

**Comment by AMIDIBOSS** [comment]
We already have poles with suspended wires. It shall not be hard to add bundled cables. But I think this will have to wait till 1.10 port.

**Comment by pooing1234** [comment]
@AMIDIBOSS How long could that be?

**Comment by AMIDIBOSS** [comment]
No idea. You shell ask @Baughn . He's working on it. I think...

**Comment by pooing1234** [comment]
@AMIDIBOSS Thanks. Oh btw, what does the grid transformer look like? any screenshots?

**Comment by AMIDIBOSS** [comment]
You can actually use them: https://github.com/Electrical-Age/ElectricalAge/tree/feature/t2-poles?files=1
But you have to compile it yourself. And dont use it on your main world, or any you care about. Create another world. I think you'll like it

**Comment by AMIDIBOSS** [comment]
And if you cant compile it youself, maybe i can do this for you if i have some time today...

**Comment by pooing1234** [comment]
@AMIDIBOSS I'm unsure how to compile the mod with gradle. But if you could send a copy of the jar file over, that would mean a lot.

But could you also point me to some sort of tutorial on how to do it?

**Comment by AMIDIBOSS** [comment]
That's a little to complicated to explain here, but... Here! Have a file!
[ElectricalAge-1.14.2-t2poles.jar.zip](https://github.com/Electrical-Age/ElectricalAge/files/994103/ElectricalAge-1.14.2-t2poles.jar.zip)

**Comment by AMIDIBOSS** [comment]
And the readme file contains "Building from source" chapter. You should refer to that:
https://github.com/Electrical-Age/ElectricalAge/blob/develop/README.md

**Comment by pooing1234** [comment]
@Thank you so much!

**Comment by pooing1234** [comment]
@AMIDIBOSS Thank you so much!

**Comment by pooing1234** [comment]
@AMIDIBOSS Doesn't seam to work. comes up with an error.

**Comment by AMIDIBOSS** [comment]
What error is this? May you give me a log?

**Comment by pooing1234** [comment]
@AMIDIBOSS https://1drv.ms/u/s!Ahm7U3S2j-ABv0HyTUrXX_Tabn72

**Comment by AMIDIBOSS** [comment]
I don't know what happend. It's a problem with my build, that's for sure. emm...

*But it works on my machine!*

**Comment by cm0x4D** [comment]
@pooing1234: Maybe a stupid question, but did you remove the release jar before adding the custom build?

**Comment by pooing1234** [comment]
@cm0x4D Yep, I removed the older electrical age build, and even done a fresh reinstall of minecraft.

**Comment by pooing1234** [comment]
@AMIDIBOSS Mostly, I just want to know what the grid transformer looks like. :)

**Comment by angelwork727** [comment]
@pooing1234 I can give you a screenshot of the grid transformer! But I am not sure if I should upload it or not...

**Comment by pooing1234** [comment]
@ihaveaquestion02 Go a head. After all, it's open source.

**Comment by angelwork727** [comment]
@pooing1234 Okay! The first pic shows the transformer itself, the second pic shows how it needs to be connected! The large towers are transmission towers, don't know if you have seen them already... 
![2017-05-15_11 15 45](https://cloud.githubusercontent.com/assets/26504003/26078771/e11cb9f6-39c0-11e7-8bd1-22b5d87c2cfc.png)
![2017-05-15_11 26 03](https://cloud.githubusercontent.com/assets/26504003/26078777/e5aea632-39c0-11e7-813d-a3f106b423d0.png)

I also have a build of ELN where all this is functioning already....

**Comment by angelwork727** [comment]
Forgot to add : the arrangement is like this : Power source - Normal powerpole - Grid transformer - transmission tower(s) - Grid transformer - Normal powerpole - Normal transformer - Load (Lamps, oven, etc)
The Transmission towers can hold a max. voltage of 60kV I think, thats all I know about them...
Another thing : The Grid Transformer has a Temperature rating, that means if you overload it it will heat up the same way as cables do, and eventually go boom.

**Comment by pooing1234** [comment]
@ihaveaquestion02 That's pretty cool! The grid transformer could be a little bigger, but this is cool! are there any more items?

**Comment by angelwork727** [comment]
I am not exactly sure but I think the only new items are the grid transformer, the transmission tower,
I am not sure about these : a bunch of electric chips (like NOT gates and such), a fuel heat furnace (basically like the stone heat furnace but runs on fuel), a green gas turbine (like the steam turbine but runs on gasoline, is used to power the generator) and at last a fire buzzer (it buzzes instead of outputting a signal).

I do not know if any of these are already in the latest available build of ELN or if they will be added or not (I'm using a dev build, not the official release).

BTW : the grid transformer's size is actually (WxLxH) : 4x4x4.

**Comment by pooing1234** [comment]
@ihaveaquestion02 Thanks! Really hyped up for the next build. Really appreciate your help.

By bigger, I meant taller. :)

**Comment by AMIDIBOSS** [comment]
Isn't the converter 2x2x2?
I mean, I created the model, and i created it 2x2x2...
Unless there are changes i don't know about?
And it's "converter", not transformer.
![figure1b](https://cloud.githubusercontent.com/assets/10089795/26265566/e78f54c8-3ce1-11e7-9f1b-69909ba5dad5.gif)
If you're intrested.

**Comment by pooing1234** [comment]
@AMIDIBOSS Sorry, I was going by what others where calling it. But it would be taller in my opinion. also could do with a lid on the top. :)

**Comment by angelwork727** [comment]
@AMIDIBOSS Sorry, You are right, it was 2x2x2! I somehow got that wrong...

**Comment by pooing1234** [comment]
@AMIDIBOSS It's alright. Here's an idea, you could but a fan on the side. You know, for visuals! :D

**Comment by pooing1234** [comment]
@AMIDIBOSS Fans please.

**Comment by marcojensen99** [comment]
Hi, I think it looks really good with air conduits and have tried to see in your thread how to add it but I could not figure it out. And if there could help me?

**Comment by AMIDIBOSS** [comment]
Heyea. Imma working on PnS system. (Plug 'n Socket system). allowing you to conect socket to socket with cable. Similar to what you have shown. But with few more sockets...

**Comment by AMIDIBOSS** [comment]
will work fine,
I guess

**Comment by marcojensen99** [comment]
Thought if you could make it was something in it's style

https://www.google.dk/search?q=luftledninger&source=lnms&tbm=isch&sa=X&ved=0ahUKEwjrlrfVxsrUAhXBd5oKHbdMDUAQ_AUIBigB&biw=1680&bih=954#imgrc=HC9W0jZRxay4AM:

**Comment by AMIDIBOSS** [comment]
I understand...
But Baughn says "No, for now..."
